client/src/modules/authorization/components/LinkExpired.jsx

```diff
@@ -16,11 +20,22 @@ class LinkExpired extends PureComponent {
   handleResendLink = () => {
     const {
       match: {
-        params: { hash }
+        params: { token },
+        path
       }
     } = this.props;
 
-    return resendConfirmationLink(hash)
+    let action;
+
+    if (path.includes(Routes.CONFIRMATION_LINK_EXPIRED)) {
```

Comment:
proposal: maybe use else or switch to prevent checking both if statements every time

Reply:
how can I use switch here?

Reply:
Switch: my mistake. For me, if else would be better than to ifs but leave as it is if you want